<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border_radius:50px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius:50px;  </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Login_Btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1c1c1e;
 color: white;
  border-radius: 10px;
 font-weight: bold;
 font-size:25px;


</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>651</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="Account">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f2f2f7;
 border: none;
 border-radius: 8px;
 padding: 10px;
  font-size: 20px;
QLineEdit {
    /* 输入文字样式 */
    font-size: 20px;                    /* 输入文字变大 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, Arial;
    color: black;
    padding: 10px 15px;
    border: 2px solid #cccccc;
    border-radius: 6px;
    background-color: white;
    min-height: 44px;
}

/* 提示文字（placeholder）样式 */
QLineEdit::placeholder {
    font-size: 20px;                    /* 提示文字也变大 */
    color: #999999;
    font-style: italic;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Code">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f2f2f7;
 border: none;
 border-radius: 8px;
 padding: 10px;
  font-size:20px;
QLineEdit {
    /* 输入文字样式 */
    font-size: 20px;                    /* 输入文字变大 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, Arial;
    color: #333333;
    padding: 10px 15px;
    border: 2px solid #cccccc;
    border-radius: 6px;
    background-color: white;
    min-height: 44px;
}

/* 提示文字（placeholder）样式 */
QLineEdit::placeholder {
    font-size: 20px;                    /* 提示文字也变大 */
    color: #999999;
    font-style: italic;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Code_Choice_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>168</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
    background-color: transparent;
    
    /* 文字样式 */
    color: #666666;
    font-size: 20px;
    font-weight: 500;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
    
    /* 尺寸和边距 */
    padding: 12px 24px;
    min-width: 120px;
    min-height: 48px;
    
    /* 鼠标光标 */
    cursor: pointer;
    
    
}
QPushButton {
    color: rgb(0,,0,0);  /* 选中时蓝色文字 */
    font-weight: 600;
    border-bottom: 3px solid rgb(0,0,0);  /* 蓝色下划线 */
}</string>
    </property>
    <property name="text">
     <string>密码登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Btn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/close.png);
QPushButton#close_Btn:hover {
    
    transform: scale(1.1) rotate(90deg);
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="new_At_Btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
